engine/gamesys/src/gamesys/scripts/script_particlefx.cpp

```diff
@@ -106,13 +106,15 @@ namespace dmGameSystem
 
             EmitterStateChangedCallbackArgs args(data.m_ComponentId, emitter_id, emitter_state);
 
-            if (!dmScript::InvokeCallback(data.m_CallbackInfo, EmitterStateChangedCallbackArgsCB, &args))
+            int ok = dmScript::InvokeCallback(data.m_CallbackInfo, EmitterStateChangedCallbackArgsCB, &args);
+            if (!ok)
             {
-                dmLogError("Could not run particlefx callback because the instance has been deleted.");
+                dmLogWarning("Could not run particlefx callback because the instance has been deleted.");
```

Comment:
Should we log once or not at all? I get a feeling that we are not consistent with this in the code in general.

Reply:
Not sure what's best.
Also, it's a bit different for particles since they're fire-and-forget.

Reply:
Also note that the `dmScript::InvokeCallback()` is deprecated.

Reply:
I decided to not log anything. And I've changed from InvokeCallback.